sentry-android-okhttp/lint-baseline.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
No issue for this module, but for some reason, it requires the baseline, likely a bug.